backend/modelsx/progress.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 type Progress struct {
-	Progress int `json:"progress"`
+	Clips map[HashID]int `json:"clips"`
```

Comment:
Does this change cause any issues on the frontend? Do we need to change anything there?

Reply:
This MR already changes the relevant points in the frontend that depended on the previous implementation..